godtools/App/Share/Data/MobileContentAuthTokenRepository/MobileContentAuthTokenRepository.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MobileContentAuthTokenRepository.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/31/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import Foundation
+import OktaAuthentication
+import Combine
+
+class MobileContentAuthTokenRepository {
+    
+    private let api: MobileContentAuthTokenAPI
+    private let cache: MobileContentAuthTokenCache
+    
+    private var cancellables = Set<AnyCancellable>()
+    
+    init(api: MobileContentAuthTokenAPI, cache: MobileContentAuthTokenCache) {
+        
+        self.api = api
+        self.cache = cache
+    }
+    
+    func getAuthTokenPublisher(for userId: Int? = nil, oktaAccessToken: String) -> AnyPublisher<String?, URLResponseError> {
```

Comment:
I think here it would be nice if this could fallback to fetching from Remote if fetching from cache doesn't provide a value.

Reply:
It's doing that already, but should be more obvious since renaming to `fetchRemoteAuthToken()`

Reply:
Oh yeah it is !